Discussion (#88)
chhoumann/p8: report/src/bluetooth_presence_methods/choice.tex

Between the two approaches, the fingerprinting approach can easily be configured with beacons to increase accuracy.
Edge cases for this approach can be analyzed and handled by setting some threshold for when a \textit{present} label is considered correct. 
This threshold could be found experimentally or determined from existing research. 

> For instance, \citeauthor{ble_kneares_neural}~\cite{ble_kneares_neural} showed that a kNN approach had high number of measurements determined within a 3 meters precision.

Reviewer: ```suggestion
For instance, \citeauthor{ble_kneares_neural}~\cite{ble_kneares_neural} showed that a k-nearest-neighbor approach yielded a high number of measurements with an accuracy up to 3 meters.
```

> As the goal of the project is not to localize people in a room, but to classify whether they are present or not, we will use the k-nearest-neighbor method for classification.

Reviewer: ```suggestion
As the goal of the project is not to localize people in a room, but to classify whether they are present or not, we will use the kNN method for classification.
```

Between the two approaches, the fingerprinting approach can easily be configured with beacons to increase accuracy.
Edge cases for this approach can be analyzed and handled by setting some threshold for when a \textit{present} label is considered correct. 
This threshold could be found experimentally or determined from existing research. 

> For instance, \citeauthor{ble_kneares_neural}~\cite{ble_kneares_neural} showed that a kNN approach yielded a high number of measurements with an accuracy up to 3 meters.

Reviewer: ```suggestion
For instance, \citeauthor{ble_kneares_neural}~\cite{ble_kneares_neural} showed that a kNN approach yielded high precision, in a range of up to 3 meters.
```
